include/fmt/format.h

```diff
@@ -192,7 +187,9 @@ FMT_BEGIN_NAMESPACE
 namespace detail {
 // Avoid Clang with Microsoft CodeGen's -Wunknown-pragmas warning.
 #  if !defined(__clang__)
-#    pragma managed(push, off)
+#    if !defined(__ICL)
+#       pragma managed(push, off)
+#    endif
```

Comment:
Let's remove `#pragma managed` completely as we don't support managed extensions.

Reply:
So done. Please have a look.